launch/launch/actions/execute_process.py

```diff
@@ -190,6 +195,14 @@ def __init__(
                 self.__env.append((
                     normalize_to_list_of_substitutions(key),
                     normalize_to_list_of_substitutions(value)))
+        self.__additional_env = \
+            None  # type: Optional[List[Tuple[List[Substitution], List[Substitution]]]]
```

Comment:
Please use the normal style of variable typing, the comment based style was only require for Python 3.5 which we don't support any more.

Reply:
I believe it would be:

```python
self.__additional_env: Optional[List[Tuple[List[Substitution], List[Substitution]]]] = None
```